src/Filecoin/Address.cpp

```diff
@@ -16,165 +15,283 @@ namespace TW::Filecoin {
 static const char BASE32_ALPHABET_FILECOIN[] = "abcdefghijklmnopqrstuvwxyz234567";
 static constexpr size_t checksumSize = 4;
 
-bool Address::isValid(const Data& data) {
-    if (data.size() < 2) {
+/// Parses the given `string` as an ActorID.
+/// Please note `string` must not contain any prefixes.
+static bool parseActorID(const std::string& string, uint64_t& actorID) {
+    if (string.length() > 20) {
```

Comment:
what means 20 here, can we use a constant variable or constant global variable for more meanings?